.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: Create a report to help us improve 
+title: "🪲[BUG] <title>"
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the sections below to help everyone identify and fix the bug
+
+        **Contribution Guidelines**
+        - If you want to work on an issue, please let us know in a comment below.
+        - Do not begin work on the issue unless it has been assigned to you.
+        - If you do not respond in 3 days, the issue may be reassigned to someone else.
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      placeholder: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to reproduce
+      placeholder: |
+        1. Go to page '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      placeholder: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Put here any screenshots or videos (optional)
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Extra fields
```

Comment:
Perhaps this could be given a better title than "Extra Fields" since some of these fields are required?